main.go

```diff
@@ -109,7 +108,11 @@ func apiHandler(w http.ResponseWriter, req *http.Request) {
 		"path": req.URL.Path,
 	})
 	w.WriteHeader(410)
-	w.Write([]byte(apiResponse))
+	_, err := w.Write([]byte(apiResponse))
+	if err != nil {
+		log.Fatal(req.Context(), "error writing response", err)
+		os.Exit(1)
```

Comment:
I don't think this error should be fatal - why kill the server when it may be that the client/network has gone, but this app is ok?
I would argue that the error should be logged only.

```suggestion
		log.Error(req.Context(), "error writing response", err)
```

Reply:
Done